.github/workflows/build-pr.yml

```diff
@@ -19,118 +19,154 @@ jobs:
     environment: Development
     steps:
       - name: "Checkout the repo"
-        uses: actions/checkout@v2
-      
+        uses: actions/checkout@v3
+
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
 
       - name: Cache Docker layers
-        uses: actions/cache@v2
+        uses: actions/cache@v3
         with:
           path: /tmp/.docman-cache
           key: prime-docman-${{ github.sha }}
           restore-keys: |
             prime-docman-
 
       - name: "Login to OpenShift repository"
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           registry: ${{secrets.OPENSHIFT_IMAGE_REPOSITORY}}
           username: ${{secrets.OPENSHIFT_TOOLS_SERVICE_ACCOUNT_USERNAME}}
           password: ${{secrets.OPENSHIFT_TOOLS_SERVICE_ACCOUNT_PASSWORD}}
 
       - name: "Get current pull request"
         id: PR
-        uses: 8BitJonny/gh-get-current-pr@1.0.1
+        uses: 8BitJonny/gh-get-current-pr@2.2.0
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
           filterOutClosed: true
 
-      - name: "Build, tag and push Document Manager image"
-        uses: docker/build-push-action@v2
-        id: build-image
-        with: 
-          context:  document-manager/backend
-          push: true
-          file: document-manager/backend/openshift.dockerfile
-          tags: ${{secrets.OPENSHIFT_IMAGE_REPOSITORY}}/${{secrets.OPENSHIFT_LICENSE_PLATE}}-tools/prime-document-manager:pr-${{steps.PR.outputs.number}}
-          build-args: |
-            BUILD_LOGLEVEL=2
-            SVC_NAME=pr-${{steps.PR.outputs.number}}
-          cache-from: type=local,src=/tmp/.docman-cache
-          cache-to: type=local,dest=/tmp/.docman-cache-new,mode=max
-
-      - name: Move cache
+      # - name: "Build, tag and push Document Manager image"
+      #   uses: docker/build-push-action@v3
+      #   id: build-image
+      #   with:
+      #     context:  document-manager/backend
+      #     push: true
+      #     file: document-manager/backend/openshift.dockerfile
+      #     tags: ${{secrets.OPENSHIFT_IMAGE_REPOSITORY}}/${{secrets.OPENSHIFT_LICENSE_PLATE}}-tools/prime-document-manager:pr-${{steps.PR.outputs.number}}
+      #     build-args: |
+      #       BUILD_LOGLEVEL=2
+      #       SVC_NAME=pr-${{steps.PR.outputs.number}}
+      #     cache-from: type=local,src=/tmp/.docman-cache
+      #     cache-to: type=local,dest=/tmp/.docman-cache-new,mode=max
```

Comment:
Can you please remove the commented lines (that will be available in Git history)?